cypress/plugins/index.js

```diff
@@ -0,0 +1,25 @@
+/// <reference types="cypress" />
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+/**
+ * @type {Cypress.PluginConfig}
+ */
+ const { initPlugin } = require('cypress-plugin-snapshots/plugin');
+
+module.exports = (on, config) => {
+  initPlugin(on, config);
+  return config;
+};
```

Comment:
Can you remove the semi-colons as it is not needed? and fix the indentation as it will show as errors with [Standard](https://standardjs.com/)library but work fine with or with out it

Reply:
`standard --fix` should auto-fix the formatting issues @mcrwong